src/main/java/frc/robot/constants/AimingConstants.java

```diff
@@ -31,36 +31,36 @@ public enum AimState {
     }
     
     // Teleop Constant
-    public static final double MAX_ELEVATOR_TELEOP_INCREMENT = 0;
+    public static final double MAX_ELEVATOR_TELEOP_INCREMENT = 0.05;
     public static final double MAX_WRIST_TELEOP_INCREMENT = 0;
     
     
     // PID Constants
-    public static final PIDParameters mElevatorPIDConstants = new PIDParameters(0.1, 0, 0);
+    public static final PIDParameters mElevatorPIDConstants = new PIDParameters(3, 0, 0);
     public static final PIDParameters mWristPIDConstants = new PIDParameters(0.1, 0, 0, 0, 0);
     public static final double WRIST_KG = 0.0;
-    public static final double ELEVATOR_FEEDFORWARD_CONSTANT = 0;
+    public static final double ELEVATOR_FEEDFORWARD_CONSTANT = 0.05;
     public static final double SPARK_THROUGHBORE_GEAR_RATIO = 1;
 
 
     public static final AimingMotorMode INITIAL_MOTOR_MODE = AimingMotorMode.BRAKE;
 
-    public static final double ELEVATOR_ROTATIONS_TO_INCHES = 1;
-	  public static final double ELEVATOR_TOLERANCE_IN = 0;
+    public static final double ELEVATOR_ROTATIONS_TO_METERS = 0.013328525766295266;
```

Comment:
Keep the TODO and issue here because the ratio will be different on the actual elevator

Reply:
Done. Created issue #281 for the same.
Link to the issue: https://github.com/FRC-1294/frc2024/issues/281